book/src/concepts/running-the-world-with-systems.md

```diff
@@ -2,70 +2,79 @@
 
-// 2. Add the workload
+2. Add the workload
+```rust, noplaypen
 world.add_workload("Creators", CreateEmpty);
+```
 
-// 3. Run the workload (which will run its systems)
+3. Run the workload (which will run its systems)
+```rust, noplaypen
 world.run_workload("Creators");
 ```
 
-This is great because it avoids a ton of boilerplate and provides outer-parallelism without having to do anything.
-
 Adding multiple systems to a workload is only a matter of expanding the second argument to a tuple. For example: 
 
-```
-// 1. Define another system
+1. Define another system
+```rust, noplaypen
 #[system(CreateCount)]
-pub fn run (mut entities:&mut Entities, mut counts:&mut Count) {
-    entities.add_entity(&mut empties, Count(0));
-}
+fn run (entities: &mut Entities, counts: &mut Count) { ... }
+```
+
+This time we use the macro, it'll make the struct and impl for you! And no tuple, the macro will also take care of it.
 
-// 2. Add the workload
+2. Add the workload
+```rust, noplaypen
 world.add_workload("Creators", (CreateEmpty, CreateCount));
+```
 
-// 3. Run the workload
+3. Run the workload
+```rust, noplaypen
 world.run_workload("Creators");
 ```
 
+This is great because it avoids a ton of boilerplate and provides [outer-parallelism](../concepts/inner-outer-parallelism.md) without having to do anything.
+
 There's a few points to keep in mind about workloads:
 1. Workloads will run its systems in parallel where possible. If they can't be run in parallel, then systems run sequentially left-to-right.
-2. (TODO: is this true?) A workload cannot be modified once its defined. Think of it more as a one-time-setup thing than something you do dynamically at runtime.
-3. (TODO: what exactly happens?) Adding a workload with a name that already exists is an error
-4. (TODO: use-case for try_add_workload()?)
-
-So what's with the double `mut` in the system definitions (e.g. **mut** empties:&**mut** Empty)? Let's think about what we need the system to change. In this case it's for sure `entities` since we're adding a new entity, but there's more - it's actually the components too. More precisely, it's not that we're mutating the _contents_ of a specific component in this example - it's that we're adding a new component to the underlying `Storage` that contains the components. We need a mutable reference to the storage. In Shipyard terms - this means we need the system to get a `ViewMut`, e.g. a mutable _view_ into the `Storage`.
-
-With that in mind there's actually two things we need to inform the compiler so that it can do its magic:
-
-1. We need to tell Rust that we want a mutable reference to the variables. This is the first `mut`
-2. We need to tell Shipyard that we want a `ViewMut` (as opposed to a `View`). This is the second `mut`
```

Comment:
The above blocks are being moved to a new chapter, yeah? Just didn't see it in this PR ;)

Reply:
Yes, it'll be part of the next push with the two previous chapters, I have it in local